staging/src/k8s.io/cloud-provider/volume/helpers/rounding.go

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Not sure if volume rounding belongs in this package, open to other suggestions here.